api/cont.js

```diff
@@ -114,7 +114,17 @@
                 continuation
                     .extend({source: sourceFragment}, idx, sourceLength, value)
                     .evalReference();
-            });
+            };
+
+            if (typeof source.length === 'number') {
+                $.each(source, eachCallback);
+            }
+            else {
```

Comment:
So how did you think of even trying to stop using `$.each`? And also how did you know to only use it if `source.length` is undefined?